test/token/ERC721/ERC721.test.js

```diff
@@ -92,10 +92,10 @@ describe('ERC721', function () {
         expect(await this.token.tokenURI(firstTokenId)).to.be.equal(newBaseURI + sampleUri);
       });
 
-      it('token URI is empty for tokens with no URI but with base URI', async function () {
+      it('token URI is appended to base URI for tokens with no URI', async function () {
```

Comment:
```suggestion
      it('tokenId is appended to base URI for tokens with no URI', async function () {
```

Reply:
Oh, of course. I'm merging this with the other comments changes mentioned below.